src/physfs.mk

```diff
@@ -32,6 +32,18 @@ define $(PKG)_BUILD
         -lphysfs -lz
 endef
 
-$(PKG)_BUILD_x86_64-w64-mingw32 =
+define $(PKG)_BUILD_SHARED
+    cd '$(1)' && cmake . \
+        -DCMAKE_TOOLCHAIN_FILE='$(CMAKE_TOOLCHAIN_FILE)' \
+        -DPHYSFS_BUILD_SHARED=TRUE \
+        -DPHYSFS_BUILD_STATIC=FALSE \
+        -DPHYSFS_INTERNAL_ZLIB=FALSE \
+        -DPHYSFS_BUILD_TEST=FALSE \
+        -DPHYSFS_BUILD_WX_TEST=FALSE
```

Comment:
It might be better if you did something like this:

```make
       $(if $(BUILD_SHARED), \
         -DPHYSFS_BUILD_SHARED=TRUE \
         -DPHYSFS_BUILD_STATIC=FALSE, \
         -DPHYSFS_BUILD_SHARED=FALSE) \
```